### ISSUE TITLE
Datetime inconsistency between Qdrant local and remote

### ISSUE THREAD

**Issue by Anush008** [qdrant/qdrant-client#532]
Specifying datetime without a TZ works as expected when using a Qdrant server but fails with Qdrant local.

```python
from qdrant_client import QdrantClient, models

client = QdrantClient(":memory:") # client = QdrantClient("http://localhost:6333") works fine

COLLECTION_NAME = "test"

client.recreate_collection(
    COLLECTION_NAME, models.VectorParams(size=384, distance=models.Distance.COSINE)
)

client.upsert(
    COLLECTION_NAME,
    [
        models.PointStruct(
            id=1, vector=[0.1] * 384, payload={"date": "1969-07-21T20:17:40"}
        )
    ],
)

results, _next = client.scroll(
    COLLECTION_NAME,
    scroll_filter=models.Filter(
        must=[
            models.FieldCondition(
                key="date", range=models.DatetimeRange(lte="2000-09-21T20:20:30")
            )
        ]
    ),
)

print(results)
```

<details>
<summary>TRACEBACK</summary>

```console
Traceback (most recent call last):
  File "/Users/anush/Desktop/qtest/main.py", line 19, in <module>
    results, _next = client.scroll(
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/qdrant_client.py", line 900, in scroll
    return self._client.scroll(
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/qdrant_local.py", line 424, in scroll
    return collection.scroll(
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/local_collection.py", line 938, in scroll
    return self._scroll_by_id(
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/local_collection.py", line 981, in _scroll_by_id
    payload_mask = calculate_payload_mask(
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 261, in calculate_payload_mask
    if check_filter(payload_filter, payload, ids_inv[i]):
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 232, in check_filter
    if not check_must(payload_filter.must, payload, point_id):
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 202, in check_must
    return all(check_condition(condition, payload, point_id) for condition in conditions)
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 202, in <genexpr>
    return all(check_condition(condition, payload, point_id) for condition in conditions)
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 171, in check_condition
    return any(check_range_interface(condition.range, v) for v in values)
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 171, in <genexpr>
    return any(check_range_interface(condition.range, v) for v in values)
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 94, in check_range_interface
    return check_datetime_range(condition, value)
  File "/Users/anush/.pyenv/versions/3.9.5/lib/python3.9/site-packages/qdrant_client/local/payload_filters.py", line 120, in check_datetime_range
    and (condition.lte is None or dt <= condition.lte)
TypeError: can't compare offset-naive and offset-aware datetimes
```

</details>

**Comment by joein** [comment]
thanks for pointing it out

#538

**Comment by joein** [comment]
available as of `qdrant-client==1.8.1`
